policy-controller/k8s/index/src/lib.rs

```diff
@@ -63,6 +63,9 @@ pub struct ClusterInfo {
 
     /// The cluster-wide default protocol detection timeout.
     pub default_detect_timeout: time::Duration,
+
+    /// The networks that probes are expected to be from.
+    pub probe_networks: Option<Vec<IpNet>>,
```

Comment:
I think the Option isn't necessary -- what would it mean for there to be `Some(vec![])`? It's functionally equivalent.

```suggestion
    /// The networks that probes are expected to be from.
    pub probe_networks: Vec<IpNet>,
```